offline_cam/data_loading.hpp

```diff
@@ -52,7 +53,7 @@ typedef struct {
 static std::map<ullong, sensor_types> load_data() {
     const char* illixr_data_c_str = std::getenv("ILLIXR_DATA");
     if (!illixr_data_c_str) {
-        std::cerr << "Please define ILLIXR_DATA" << std::endl;
+        spdlog::get("illixr")->error("[OFFLINECAM] Please define ILLIXR_DATA");
```

Comment:
Nit: why is it `[OFFLINECAM]` here instead of something like `[offline_cam]`? Same for offline_imu, etc.

Reply:
IIRC I was originally using all caps everywhere before switching to snake case and just missed these. Fixed.